pull_request_template.md

```diff
@@ -1,48 +1,26 @@
-<!--
-## Requirements
-
-- Filling out the template is required. Any pull request that does not include enough information to be efficiently reviewed may be rejected.
-- Please consider which branch this is to be submitted against. This will normally be the development branch, so please ask your reviewer if you think it needs to go somewhere else. 
--->
-
 ## Type
-
 - [ ] Bug fix
-- [ ] New feature
-- [ ] Improvement to existing feature
-
-## Related JIRA Issue(s)
-<!--_Please provide the URL(s) for any JIRA issues related to this PR._-->
-
-## Importance
-<!-- Please explain briefly if this PR specifically needs more importance and also if there is any due date by which it needs to be merged. -->
+- [ ] New feature / Improvement
+- [ ] Housekeeping
 
 ## Description
 <!--
 _Using one or more sentences, describe the proposed changes and the reason for making them._
 -->
 
-## Deployment URL
+
+## Related JIRA Issue(s)
+<!--_Please provide the URL(s) for any JIRA issues related to this PR._-->
+
+## Deployment URL(s)
 <!--
```

Comment:
we dont these instructions anymore since there isn't anything that needs deploying. Should we remove the confluence URL?

Reply:
I've removed the comment from `Deployment URL(s)`